es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super(width);
+    this.width = width;
+    this.area = width * width;
+    this.perimeter = width * 4;
```

Comment:
You shouldn't be creating these properties manually. You should be utilizing `super` to generate these properties for you